experiments/coq-experiments/5.1/Collect.py

```diff
@@ -25,16 +23,15 @@ def collect(results: str):
             run_trial = None
 
             for strategy in tool.all_strategies(workload):
-                if strategy.name not in [
-                        'TypeBasedGenerator', 'BespokeGenerator', 'TypeBasedFuzzer',
-                        'SpecificationBasedGenerator'
-                ]:
+                if strategy.name in ['RookieGenerator']:
```

Comment:
Can you undo this change please?

Reply:
Or what is the RookieGenerator in this context?

Reply:
It's the BST style generator for RBT, we don't use it in any of our trials, it was there as an early benchmark.

Reply:
I can remove it altogether if you want. We don't use it anyway.

Reply:
Oh ok, I'd rather enumerate the strategies that we do use than the ones that we don't because I find it easier to read. (I read this backwards at first). I will push a fix.